airbyte-integrations/connectors/source-kafka/src/test/java/io/airbyte/integrations/source/kafka/AvroConverterTest.java

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.kafka;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.airbyte.integrations.source.kafka.format.Avro2JsonConvert;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AvroConverterTest {
+
+    ObjectMapper mapper = new ObjectMapper();
+
+    String avroSimpleSchema = """
```

Comment:
I wonder if we should move all this string literals to resource files to keep the test class a bit tidier.

Reply:
I agree, it's better. I'll do